CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## Unreleased
+## 0.19.0 (May 20, 2024)
```

Comment:
```suggestion
## Unreleased

## 0.19.0 (May 20, 2024)
```

I think we can leave the Unreleased title above and just add the version below. Similar to what was done [here](https://github.com/hashicorp/vault-plugin-auth-azure/pull/155/files)